### PR TITLE
Fix Heartbeat mem and cpu stats

### PR DESCRIPTION
**Pull request by alpeb**
Since k8s 1.16 cadvisor uses the `container` label instead of
`container_name` in the prometheus metrics it exposes.
The heartbeat queries were using the latter, so they were broken
for k8s version since 1.16.

Note that the `p99-handle-us` value is still missing because the
`request_handle_us` metrics is always zero.

### CODE REVIEW

**Comment by alpeb** [comment]
Raised #5043 to track the `request_handle_us` issue.

File: controller/heartbeat/heartbeat.go [hunk 1]
```diff
@@ -122,7 +121,9 @@ func PromValues(promAPI promv1.API, controlPlaneNamespace string) url.Values {
 		}
 
 		// p95-cpu
-		query = fmt.Sprintf("max(quantile_over_time(0.95,rate(container_cpu_usage_seconds_total%s[5m])[24h:5m]))", containerLabels)
+		query = fmt.Sprintf("max(quantile_over_time(0.95,rate(container_cpu_usage_seconds_total%s[5m])[24h:5m]) "+
+			"or quantile_over_time(0.95,rate(container_cpu_usage_seconds_total%s[5m])[24h:5m]))",
```
Comment (zaharidichev) [inline controller/heartbeat/heartbeat.go:125]: Was hoping there is a less verbose way, but it seems that is not the case.

**Review by zaharidichev — APPROVED** [review]
LGTM!

**Review by kleimkuhler — APPROVED** [review]
Looks good 👍

**Pull request merged**